src/App.js

```diff
@@ -24,8 +25,9 @@ function App() {
 
     return (
         <>
-            <NavBar onSubmit={handleSearchSubmit} />
-            <AdminPanel />
+            <LoginPage />
+            {/* <NavBar onSubmit={handleSearchSubmit} /> */}
+            {/* <AdminPanel /> */}
```

Comment:
Myśle że możemy to usunąć, jak będe robić nawigacje to to ogarne już na własną ręke